src/rest-client.ts

```diff
@@ -486,6 +486,9 @@ export class RestClient extends BaseRestClient {
     return this.postPrivate('/api/v5/account/position/margin-balance', params);
   }
 
+  /**
+   * @deprecated Use getLeverageV2 instead
+   */
```

Comment:
is the getLeverageV2 a left-over from the migration to object params? The type looks compatible, and it uses the same endpoint now. Did we miss removing getLeverageV2? Might be good to update the type of `getLeverage` to match the docs and remove `getLeverageV2`..